workos/organizations.py

```diff
@@ -223,6 +224,14 @@ def delete_organization(self, organization_id: str) -> None:
             method=REQUEST_METHOD_DELETE,
         )
 
+    def list_organization_roles(self, organization_id: str) -> RolesList:
+        response = self._http_client.request(
+            f"organizations/{organization_id}/roles",
+            method=REQUEST_METHOD_GET,
+        )
+
+        return RolesList.model_validate(response)
```

Comment:
We don't use WorkOSListResource, because the API isn't paginated, but perhaps it'd be more standardized if it was?